src/libs/elektra/keymeta.c

```diff
@@ -221,10 +235,11 @@ const Key * keyCurrentMeta (const Key * key)
 	return ret;
 }
 
-/**Do a shallow copy of metadata from source to dest.
+/** Do a shallow copy of metadata with name @p metaName from source to dest.
  *
  * The key dest will have the same metadata referred with
- * metaName afterwards then source.
+ * metaName afterwards then source. If the Key with name @p metaName doesn't
+ * exist in @p src - it gets deleted. Read-only keys are unaffected.
```

Comment:
Github does not let me comment properly here, because there is a line removed in between my suggestion.
Please apply this only manually, otherwise it will break the docu.

I think the **whole** paragraph should be something like:
```suggestion
 * Afterwards @p source and @p dest will have the same metadata referred with @p metaName.
 * If the Key with name @p metaName doesn't  exist in @p src - it gets deleted in @p dest.
 * Read-only keys are unaffected.
```

The current state is quite unreadable.